src/main/java/com/devoxx/genie/util/DefaultLLMSettingsUtil.java

```diff
@@ -35,7 +35,8 @@ public static boolean isApiKeyBasedProvider(ModelProvider provider) {
             provider == ModelProvider.Mistral ||
             provider == ModelProvider.Groq ||
             provider == ModelProvider.DeepInfra ||
-            provider == ModelProvider.Google;
+            provider == ModelProvider.Google ||
+			provider == ModelProvider.AzureOpenAI;
```

Comment:
Can you replace the tabs with spaces? Maybe this applies to all files, did not check it.

Reply:
Good suggestion, will put it on my list